package.json

```
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "autoprefixer": "^10.2.5",
+    "hugo-extended": "^0.83.1",
+    "postcss": "^8.2.13",
+    "postcss-cli": "^8.3.1"
+  }
+}
```

Since we no longer load dependencies globally in the deploy script, we need to also include `redoc-cli` as a dependency. Version `0.9.13` is the only version I've found to consistently generate the API docs.

```suggestion
{
  "devDependencies": {
    "autoprefixer": "^10.2.5",
    "hugo-extended": "^0.83.1",
    "postcss": "^8.2.13",
    "postcss-cli": "^8.3.1",
    "redoc-cli": "0.9.13"
  }
}
```